tap/extensions/amqp/main.go

```diff
@@ -74,13 +74,13 @@ func (d dissecting) Dissect(b *bufio.Reader, reader api.TcpReader, options *api.
 	var lastMethodFrameMessage Message
 
 	for {
-		frame, err := r.ReadFrame()
+		frameVal, err := r.readFrame()
```

Comment:
imported module with name frame exists (name collision)

Reply:
@gadotroee what's module that it collides with?

Reply:
I don't see any module that has this name so maybe it was mistake in the comment, I will check again now

Reply:
I think it is this one - https://github.com/up9inc/mizu/blob/develop/tap/extensions/amqp/types.go#L255 (here it is not a module but a struct with this name (in the same package)